source/_integrations/adax.markdown

```diff
@@ -18,14 +18,18 @@ Integrates Adax heater into Home Assistant.
 
 You can configure the heaters to use a local or cloud interface.
 
-The local integrations only works with newer Adax heaters with both Bluetooth and wifi. Home Assistant uses Bluetooth to configure the heaters. For the local integration, you have to reset the heater by pressing + and OK until the display shows 'Reset'. Then press and hold the OK button on the heater until the blue LED starts blinking before pressing Submit. Configuring the heater might take some minutes. Using the local integration will disable cloud communication and the Adax app will not work.
+The local integrations only works with newer Adax heaters with both Bluetooth and wifi. Home Assistant uses Bluetooth LE to configure the heaters, this means your machine running “Home Assistant” needs to have a Bluetooth adapter and the heater needs to be in range. For the local integration, you have to reset the heater by pressing + and OK until the display shows 'Reset'. Then press and hold the OK button on the heater until the blue LED starts blinking before pressing Submit. Configuring the heater might take some minutes. Using the local integration will disable cloud communication and the Adax app will not work.
```

Comment:
```suggestion
The local integrations only works with newer Adax heaters with both Bluetooth and wifi. Home Assistant uses Bluetooth LE to configure the heaters, this means your machine running “Home Assistant” needs to have a Bluetooth adapter and the heater needs to be in range during setup. For the local integration, you have to reset the heater by pressing + and OK until the display shows 'Reset'. Then press and hold the OK button on the heater until the blue LED starts blinking before pressing Submit. Configuring the heater might take some minutes. Using the local integration will disable cloud communication and the Adax app will not work.
```

Reply:
I found an unused esp32 board in my drawer this morning, I might test the bluetooth proxy idea first, as this could be a useful thing to have as heaters could be installed quite far away from the HA machine. Otoh I would expect that people who know about BT proxies would get the "in range" bit anyway.